app/views/podcasts/show.rss.builder

```diff
@@ -82,11 +82,11 @@ xml.rss "xmlns:atom" => "http://www.w3.org/2005/Atom",
       xml.itunes(:summary) { xml.cdata!(itunes_summary(@podcast)) }
     end
 
-    xml.itunes :keywords, @podcast.keywords.join(",") unless @podcast.keywords.blank?
+    # xml.itunes :keywords, @podcast.keywords.join(",") unless @podcast.keywords.blank?
```

Comment:
TODO: do we want to keep rendering these fields - maybe with strict-sanitized (lowercased etc) content?

Or do we just drop support, since they seem deprecated anyways?

Reply:
Okay, so my vote is to drop the fields from the RSS.  We can always bring them back later if it turns out folks are using this.

But to date, we haven't been setting `itunes:keywords` from our UI, so we generally don't have data to display here.  Only if it came in on an RSS import.

Reply:
I vote drop from the RSS, yes, we can bring them back if it turns our to be important